### DISCUSSION TITLE
wostreaming.net is now amperwave.net

### DISCUSSION THREAD

**Discussion by FurryFennec** [junguler/m3u-radio-music-playlists#25] [General]
First of all, thank you for this awesome resource! I had been looking for a list like this for a while to listen to some local stations without need of iHeart, Audacy, or TuneIn or using a browser. 

The two or three local stations were hosted via **live.wostreaming.net** which stopped working sometime last week. I have now found that substituting **live.amperwave.net** in its place gets me the streams now. Thought I'd pass it along and perhaps there's someway you can verify that the 400+ stations from live.wostreaming.net work from live.amperwave.net as well and you can update the list.

**Comment by junguler** [comment]
hello, thanks for the kind words

are these streams dispersed throughout the whole repo or are there are mostly in a few folders? i ask because this is a very easy task but it'll take a while to go thru the whole repo and replace links in every folder

regardless, i'll try to fix the links soon

**Comment by junguler** [comment]
hello again, i've just pushed an update to the repo that fixes links in everything-full-repo and everything-lite-repo files, these are 2 of the most viewed files in this repo and the fixes should be enough, i've decided against going thru all the files and pushing a huge update because if someone is faced with the same issue i can easily point him/her to this discussion page

thanks again for making this discussion and please don't hesitate to pointing other issues or inconsistencies wherever you saw them
